modules/shared/components/atoms/checkbox/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import type { FC, ReactElement } from 'react';
+import classNames from 'classnames';
+import type { ICheckbox } from './ICheckbox';
+import CheckBoxSvg from './CheckBoxSvg';
+
+const CHECKBOX = {
+  SIZE: {
+    DEFAULT: 'Default',
+    SMALL: 'Small',
+  },
+};
```

Comment:
checkout [this pr](https://github.com/m3ntorship/pickify-frontend/pull/71) and use enums in a seperated file instead of constants in the component file